packages/client/integration/DirectLocOpen.ts

```diff
@@ -0,0 +1,141 @@
+import { State, initRequesterBalance, TEST_LOGION_CLIENT_CONFIG } from "./Utils.js";
+import {
+    HashOrContent,
+    ItemsParams,
+    MergedFile,
+    AddFileParams,
+    MergedMetadataItem,
+    AddMetadataParams, AddLinkParams, MergedLink, LocData
+} from "../src/index.js";
+import { UUID } from "@logion/node-api";
+
+export async function openIdentityLoc(state: State): Promise<UUID> {
+    const { directRequesterAccount, signer, alice } = state;
+    const client = state.client.withCurrentAddress(directRequesterAccount);
+    let locsState = await client.locsState();
+
+    await initRequesterBalance(TEST_LOGION_CLIENT_CONFIG, signer, directRequesterAccount.address);
+
+    const items = provideItems("identity", []);
+    const openLoc = await locsState.openIdentityLoc({
+        description: "Direct Identity",
+        legalOfficer: alice,
```

Comment:
```suggestion
        legalOfficer: client.legalOfficers.find(llo => llo.address === alice.address)!,
```

Indeed, the `LegalOfficerClass` instance referenced by `alice` does not have any token set as it comes from the client without any selected current address.

Reply:
Good catch, thanks !
As discussed, to make the code more robust, the legal officer is now referenced with its address - the correct instance of `LegalOfficerClass` instance being searched within the client code.